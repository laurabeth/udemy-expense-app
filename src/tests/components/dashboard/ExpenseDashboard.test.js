/* eslint-disable no-undef */
describe("expense dashboard", () => {
  it("renders without crashing", () => {});
});
