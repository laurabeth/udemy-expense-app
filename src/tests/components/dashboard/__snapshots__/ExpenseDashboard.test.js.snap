// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expense dashboard renders without crashing 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
